src/cart/crt_hg_proc.c

```diff
@@ -429,6 +424,11 @@ crt_proc_corpc_hdr(crt_proc_t proc, struct crt_corpc_hdr *hdr)
 		D_ERROR("crt proc error, rc: %d.\n", rc);
 		D_GOTO(out, rc);
 	}
+	rc = crt_proc_crt_group_id_t(hg_proc, &hdr->coh_grpid);
```

Comment:
any particular reason to move this proc?  now the order doesn't match the field order in struct crt_corpc_hdr

Reply:
This is the new string group id. We have removed the int group id. I did not know it had to be in the same order as the struct. Will move it, change copyright and resubmit